styles/global.scss

```diff
@@ -32,6 +33,7 @@ body {
     margin: 0;
     font-size: 100%;
     color: var(--text-color);
+    background: var(--page-bg);
     line-height: 1.5em;
     -webkit-tap-highlight-color: transparent;
     scroll-behavior: smooth;
```

Comment:
why you have used em here ?

whats the webkit doing here?

Reply:
em can be used for line-height and the webkit is for mobile screen